vermouth/processors/do_mapping.py

```diff
@@ -631,7 +636,7 @@ def do_mapping(molecule, mappings, to_ff, attribute_keep=(), attribute_must=(),
                 if attr in attribute_keep or attr not in graph_out.nodes[out_idx]:
                     graph_out.nodes[out_idx].update(new_attrs)
                 if attr in attribute_stash:
-                    graph_out.nodes[out_idx][attr+"_old"] = val
+                    graph_out.nodes[out_idx]["_old_"+attr] = val
```

Comment:
Should we check `"_old_"+attr` doesn't already exist?

Reply:
I think it is ok to overwrite if it is already present. It's just stashed attributes and therefore not really a hard attribute. I think the expected behavior is pretty clear from the processor doc.